packages/docs/content/components/alert.mdx

```diff
@@ -277,62 +172,178 @@ When using a screen reader (NVDA, JAWS, VoiceOver, TalkBack):
+
+</ThemeContent>
+
+<StorybookExample componentName="error alert" storyId="components-alert--error" />
+
+### Warning
+
+<ThemeContent onlyThemes={['healthcare']}>
+
+Informs users that their entry or selection may have unexpected or undesirable results, some of which may not be easily undone. Highlights information a user should be aware of in order to help them avoid taking an action that is likely to result in a negative outcome or understand when content/data is missing from the page.
+
+</ThemeContent>
+
+<ThemeContent onlyThemes={['medicare']}>
+
+Requires an icon.
+
+Informs users that their entry or selection may have unexpected or undesirable results, some of which may not be easily undone.
+
+</ThemeContent>
+
+<StorybookExample componentName="warning alert" storyId="components-alert--warning" />
+
+### Success
+
+<ThemeContent onlyThemes={['healthcare']}>
+
+Confirms the successful completion of a step or task, or receipt of information provided by the user. Suggests a path forward that may be beneficial to the user, based on their selections or entries.
+
+</ThemeContent>
+
+<ThemeContent onlyThemes={['medicare']}>
+
+Does not require an icon.
+
+Confirms the successful completion of a step or task, or receipt of information provided by the user.
+
+</ThemeContent>
 
-### Style customization
+<StorybookExample componentName="success alert" storyId="components-alert--success" />
 
-The following CSS variables can be overridden to customize Alert components:
+<ThemeContent onlyThemes={['medicare']}>
 
-<ComponentThemeOptions componentname="alert" />
+### Note
 
-### Analytics
+Not in the design system yet.
+
+Does not require an icon.
+
+Explains concepts which may include math or maybe not.
+
+</ThemeContent>
+
+### Lightweight
+
+<ThemeContent onlyThemes={['medicare']}>
+
+<Badge variation="alert">
+  <CloseIconThin /> Unused by Medicare.
+</Badge>
+
+</ThemeContent>
+
+<ThemeContent onlyThemes={['healthcare']}>
+
+Used to show important but not as urgent messages as standard alerts.
+
+</ThemeContent>
+
+<ThemeContent neverThemes={['medicare']}>
+
+<StorybookExample componentName="lightweight alert" storyId="components-alert--lightweight" />
+
+</ThemeContent>
+
+<MaturityChecklist items={{
+  a11yStandards: true,
+  color: true,
+  forcedColors: true,
+  screenReaders: true,
+  keyboardNavigable: null,
+  storybook: true,
+  responsive: true,
+  spanish: true,
+  completeUiKit: false,
+  responsiveUiKit: false,
+  tokensInCode: true,
+  tokensInSketch: true,
+}}
+>
+
+## Component maturity
+
+</MaturityChecklist>
+
+## Learn more
+
+[18F Content Guide \- Active voice](https://content-guide.18f.gov/active-voice/)
+
+## Resources
+
+### Design
+
+<DesignResourceLink />
+
+### Development
 
 This component has analytics tracking available. Please see our developer documentation about [using analytics in the design system](/getting-started/for-developers/#analytics).
```

Comment:
I wasn't sure where our analytics content should live, so I added it under the Development section